docs/user/04-metrics.md

```diff
@@ -331,53 +342,53 @@ spec:
         value: https://backend.example.com:4317
 ```
 
-The agent configures the [kubletstatsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver) for the metric groups `pod` and `container`. With that, [system metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/documentation.md) related to containers and Pods are collected by default.
+The agent configures the [kubletstatsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver) for the metric groups `pod` and `container`. With that, [system metrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/kubeletstatsreceiver/documentation.md) related to containers and Pods are collected.
 
 If you want to disable the collection of the Pod or container metrics, define the `resources` section in the `runtime` input.
 
-The following example drops the runtime Pod metrics and only collects runtime container metrics:
+- The following example drops the runtime Pod metrics and only collects runtime container metrics:
 
-```yaml
-apiVersion: telemetry.kyma-project.io/v1alpha1
-kind: MetricPipeline
-metadata:
-  name: backend
-spec:
-  input:
-    runtime:
-      enabled: true
-      resources:
-        pod:
-          enabled: false
-  output:
-    otlp:
-      endpoint:
-        value: https://backend.example.com:4317
-```
-
-The following example drops the runtime container metrics and only collects runtime Pod metrics:
-
-```yaml
-apiVersion: telemetry.kyma-project.io/v1alpha1
-kind: MetricPipeline
-metadata:
-  name: backend
-spec:
-  input:
-    runtime:
-      enabled: true
-      resources:
-        container:
-          enabled: false
-  output:
-    otlp:
-      endpoint:
-        value: https://backend.example.com:4317
-```
+  ```yaml
+  apiVersion: telemetry.kyma-project.io/v1alpha1
+  kind: MetricPipeline
+  metadata:
+    name: backend
+  spec:
+    input:
+      runtime:
+        enabled: true
+        resources:
+          pod:
+            enabled: false
```

Comment:
Isn't one flavor of this example enough? Maybe lets just list the "container" with "enabled" true and delete the next example, the rest people can combine?
```suggestion
          pod:
            enabled: false
          container:
            enabled: true
```

Reply:
let's do this in another PR

Reply:
not sure, I would expect a call to explain why you are not interested in my feedback :P